src/Auth/Utility/invitation-forgot-password.html

```diff
@@ -0,0 +1,11 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+</head>
+<body>
+    <p>Your new password is "'Password'".</p>    
+    <p>Please use this <a href="http://docker-hip.cs.upb.de/login/?email={email}">Link to login with your new password</a></p>
```

Comment:
We should send an German mail. 

"Dein neues Password ist: "'Password'"
Unter <a....>HiP-Cms<a/> kannst du dich jetzt mit deinem neuem Password anmelden"